NuGet.config

```diff
@@ -7,8 +7,7 @@
   <packageSources>
     <add key="Nuget" value="https://www.nuget.org/api/v2/" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="GovNotify" value="https://api.bintray.com/nuget/gov-uk-notify/nuget" />
-    <add key="local-packages" value=".\Lib"/>
+      <add key="Local" value="%BUILD_SOURCESDIRECTORY%/Lib" /><!-- Allows us to put a GovUkDesignSystem package in /Lib -->

```

Comment:
Locally, `%BUILD_SOURCESDIRECTORY%` defaults to an empty string. On Azure Devops, it defaults to where it should so the pipeline does not fail. This can be fixed once / if the design library is on nuget...